wezterm-gui/src/glyphcache.rs

```diff
@@ -889,20 +889,21 @@ impl<T: Texture2d> GlyphCache<T> {
                 let pixel = buffer.pixel_mut(x, y);
                 let (current, _, _, _) = SrgbaPixel::with_srgba_u32(*pixel).as_rgba();
                 let value = current.saturating_add(val);
-                *pixel = SrgbaPixel::rgba(value, value, value, 0xff).as_srgba32();
+                *pixel = SrgbaPixel::rgba(0xFF, 0xFF, 0xFF, val).as_srgba32();
```

Comment:
```suggestion
                *pixel = SrgbaPixel::rgba(value, value, value, value).as_srgba32();
```

I think we probably want this, which is closer to the original logic, but also sets the alpha channel value as a pre-multiplied value